com.microsoft.mrtk.input/Subsystems/SyntheticHands/SyntheticHandsSubsystem.cs

```diff
@@ -52,28 +53,89 @@ private static void Register()
+
+        /// <summary>
+        /// Requests the selection pose for the specified hand.
+        /// </summary>
+        /// <param name="handNode">The hand for which the selection pose is being requested.</param>
+        /// <returns>
+        /// Identifier representing the hand pose.
+        /// </returns>
+        [Obsolete("Please use the GetSelectionHandshape(handNode) instead.")]
+        public HandshapeId GetSelectionPose(XRNode handNode) => GetSelectionHandshape(handNode);
+
+        /// <summary>
+        /// Sets the selection pose for the specified hand.
+        /// </summary>
+        /// <param name="handNode">The hand for which the selection pose is being set.</param>
+        /// <param name="poseId">The desired hand pose.</param>
+        [Obsolete("Please use the SetSelectionHandshape(handNode, handshapeId) instead.")]
+        public void SetSelectionPose(XRNode handNode, HandshapeId poseId) => SetSelectionHandshape(handNode, poseId);
+
+        /// <summary>
+        /// Requests the neutral handshape for the specified hand.
+        /// </summary>
+        /// <param name="handNode">The hand for which the neutral handshape is being requested.</param>
+        /// <returns>
+        /// Identifier representing the hand handshape.
+        /// </returns>
+        public HandshapeId GetNeutralHandshape(XRNode handNode)
+        {
+            return (provider as SynthesisProvider).GetNeutralHandshape(handNode);
+        }
+
+        /// <summary>
+        /// Sets the neutral handshape for the specified hand.
+        /// </summary>
+        /// <param name="handNode">The hand for which the neutral handshape is being set.</param>
+        /// <param name="handshapeId">The desired hand handshape.</param>
+        public void SetNeutralHandshape(XRNode handNode, HandshapeId handshapeId)
         {
-            (provider as SynthesisProvider).SetNeutralPose(handNode, poseId);
+            (provider as SynthesisProvider).SetNeutralHandshape(handNode, handshapeId);
+        }
+
+        /// <summary>
+        /// Requests the selection handshape for the specified hand.
+        /// </summary>
+        /// <param name="handNode">The hand for which the selection handshape is being requested.</param>
+        /// <returns>
+        /// Identifier representing the hand handshape.
+        /// </returns>
+        public HandshapeId GetSelectionHandshape(XRNode handNode)
+        {
+            return (provider as SynthesisProvider).GetSelectionHandshape(handNode);
+        }
+
+        /// <summary>
+        /// Sets the selection handshape for the specified hand.
+        /// </summary>
+        /// <param name="handNode">The hand for which the selection handshape is being set.</param>
+        /// <param name="handshapeId">The desired hand handshape.</param>
+        public void SetSelectionHandshape(XRNode handNode, HandshapeId handshapeId)
```

Comment:
Looks like this is still using the "selection" verbiage, which I think we changed to "trigger" elsewhere.

Reply:
I think because of how the synthetic hands explicitly intends to lerp between the two poses based on the "selectedness" value, we want to keep it as Selection in this class. The use of "trigger" in other places is in reference to how its directly associated with the "trigger" input action.